unitTests/ui/QueryBuilderTest.ts

```diff
@@ -174,5 +176,39 @@ export function QueryBuilderTest() {
     it('should be able to detect if no end user keywords exist in either the basic or long expression', () => {
       expect(queryBuilder.containsEndUserKeywords()).toBeFalsy();
     });
+
+    it('should have facets undefined if none have been added', () => {
+      expect(queryBuilder.build().facets).toBeUndefined();
+    });
+
+    it('should have facets undefined if none have been added', () => {
```

Comment:
This test looks to be a duplicate of the one on line 180.

Reply:
How?????